api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -86,4 +100,8 @@ message QuicProtocolOptions {
   // If not specified, no debug visitor will be attached to connections.
   // [#extension-category: envoy.quic.connection_debug_visitor]
   core.v3.TypedExtensionConfig connection_debug_visitor_config = 11;
+
+  // Configure a type of UDP cmsg to pass to listener filters via QuicReceivedPacket.
+  // If not specified, no cmsg will be saved to QuicReceivedPacket.
+  SaveCmsgConfig save_cmsg_config = 12;
```

Comment:
This should probably be defined as a repeated field (adda a validator ensuring that only 1 element appears if the code only supports that - if this won't be defined as repeated now and needed in the future it will require a new field).

Reply:
Done, with validation.